releasenotes/releasenotes.rst

```diff
@@ -36,6 +36,7 @@ Feature
 #. We introduced endpoints for healthchecks, both liveness and readiness, see :ref:`feature_healthcheck`.
 #. We added support for the X-Provenance header, see :ref:`feature_x-provenance`.
 #. The reference Binary.securityContext can now be used as a link for the Patient compartment. If SMART on FHIR is enabled and a "patient"-level scope is being used, Binary resources must now point to a Patient resource matching the "patient" claim in the access token. For facade and plugin implementations this change means that the ICompartment information is now available for requests on the "Binary" resource type.
+#. We enhanced audit logging by exposing the canonical references of the results from a search request. For file logging, this information can be included by adding SearchResultSummary to the output template. In AuditEvent resources, each search result (reference) is included as an ``entity`` node. 
```

Comment:
* We expose the literal references, these are not canonical references
* I would clarify that we only expose the references from the first page of the search bundle
* I would also add that they query params are exposed in the entity representing the search bundle

Reply:
Updated :-)